libevm/VM.cpp

```diff
@@ -715,6 +727,20 @@ void VM::interpretCases()
 		}
 		NEXT
 
+		CASE(RETURNDATACOPY)
+		{
+			if (!m_schedule->haveReturnData)
+				throwBadInstruction();
+
+			m_copyMemSize = toInt63(m_SP[2]);
+			updateMem(memNeed(m_SP[0], m_SP[2]));
+			ON_OP();
+			updateIOGas();
+
+			copyDataToMemory(&m_returnData, m_SP);
```

Comment:
Not strictly connected to this PR, but probably it would be better to have the offset + size as explicit parameters in `copyDataToMemory`.